CMakeLists.txt

```diff
@@ -256,6 +256,11 @@ endif()
 
 option(SQLITE3_SUPPORT "Activate SQLITE3 support" ON)
 
+###############################################################################
+# SHM as Default transport
+###############################################################################
+option(SHM_TRANSPORT_DEFAULT "Adds SHM transport to the default transports" ON)
```

Comment:
I think we should leave this OFF and set it ON on a separate PR.

We can set it ON manually on the CI builds if we want them to check the behavior with SHM on by default.